pkg/config/config.go

```diff
@@ -248,8 +268,65 @@ func CustomHTTPErrorHandler(err error, c echo.Context) {
 	}
 }
 
-func ConfigureEcho() *echo.Echo {
-	e := echo.New()
+func getResource(path string) string {
```

Comment:
What is the function for? I can't figure it out :)

Reply:
It was intended to retrieve the main resource of the API request: `repositories` for `/api/v1/repositories` or `/api/v1/repositories/{uuid}` or `/beta/api/v1/repositories/{uuid}`
but I have to update it; thanks mate